models/archipelago/ethereum/archipelago_ethereum_sources.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+sources:
+  - name: archipelago_ethereum
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
```

Comment:
Could you delete the error line here please ? Safer just to be warned for now in case the underlying table isn't refreshed